src/app/mytools/my-tool/my-tool.component.html

```diff
@@ -26,7 +26,7 @@
         <mat-sidenav-container autosize class="sidebar-set-height">
           <mat-sidenav #sidenav mode="side" [(opened)]="showSidebar" class="col-sm-3 col-xs-12">
             <span class="visible-xs" *ngIf="refreshMessage">
-              <mat-progress-bar mode="indeterminate"></mat-progress-bar><br />
+              <mat-progress-bar mode="indeterminate" class="m-2"></mat-progress-bar>
```

Comment:
check if this causes the bounce back

Reply:
if it does, copy the progress bar from other components

Reply:
I meant to put this on the other one, I reverted this back.